.travis.yml

```diff
@@ -0,0 +1,34 @@
+sudo: required
+
+before_install:
+    - sudo apt-add-repository -y ppa:ubuntu-toolchain-r/test
+    - sudo apt-add-repository -y ppa:beineri/opt-qt542
+    - sudo apt-get -qq update
+    - sudo apt-get -qq install g++-4.8 libc6-i386 qt54tools qt54svg qt54webkit qt54serialport

```

Comment:
No, because we don't have access to the files after a travis build, only the logs.